site/class-1-play/@RyanNL1994 play opdracht.svg

```diff
@@ -0,0 +1,261 @@
+<svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
You’re missing a `viewBox` attribute on your `<svg>` element!

You can edit files through the github interface, by going to your fork, your branch, and the finding your file. Finally, click on the pencil icon in the top-right.